atomicfu-gradle-plugin/src/main/kotlin/kotlinx/atomicfu/plugin/gradle/AtomicFUGradlePlugin.kt

```diff
@@ -85,23 +105,34 @@ private fun Project.configureDependencies() {
             getAtomicfuDependencyNotation(Platform.JS, version)
         )
         dependencies.add(TEST_IMPLEMENTATION_CONFIGURATION, getAtomicfuDependencyNotation(Platform.JS, version))
-        addCompilerPluginDependency()
+        addJsCompilerPluginRuntimeDependency()
     }
     withPluginWhenEvaluatedDependencies("kotlin-multiplatform") { version ->
+        addJsCompilerPluginRuntimeDependency()
         configureMultiplatformPluginDependencies(version)
     }
 }
 
-private fun Project.configureTasks() {
-    val config = config
-    withPluginWhenEvaluated("kotlin") {
-        if (config.transformJvm) configureJvmTransformation()
+private fun Project.configureMultiplatformPluginDependencies(version: String) {
```

Comment:
The only conceptual change in the `AtomicfuGradlePlugin` is in `configureMultiplatformPluginDependencies`, all other changes are just refactoring: changing the order of declarations.